src/Catrobat/Responses/ApiResponse.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+
+namespace App\Catrobat\Responses;
+
+
+use App\Catrobat\Services\Formatter\ElapsedTimeStringFormatter;
+use App\Entity\Program;
+use App\Entity\ProgramManager;
+use Psr\Container\ContainerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+
+class ApiResponse extends AbstractController
```

Comment:
You should use the already defined model as the response, no need for a new class.    -> `vendor/catrobat/capi/Model/Project.php`